hail/python/hail/linalg/blockmatrix.py

```diff
@@ -2099,7 +2099,7 @@ def _ndarray_to_makearray(ndarray):
         data = [x for row in data for x in row]
 
     data_as_ir = [F64(x) for x in data]
```

Comment:
noticed this as a drive by. If `data` is at all big, then we *definitely* don't want to be using a MakeArray here -- we should be using `Literal`

Reply:
Ok good to know. Data shouldn't be too big since it should be small values from numpy, but also an easy switch.